lib/thredded/email_notifier.rb

```diff
@@ -0,0 +1,53 @@
+module Thredded
+  class EmailNotifier
+
+    def new_post(post, users)
+      @post = post
+      users_opted_in = exclude_those_opting_out_of_followed_activity_notifications(users)
```

Comment:
For now, I would pass post to this method and not assign post to @post (shared state that is confusing)